lib/game.rb

```diff
@@ -0,0 +1,89 @@
+require 'pry'
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  
+
+  WIN_COMBINATIONS = [
+    [0,1,2], #top row
+    [3,4,5], #middle row
+    [6,7,8], #bottom row
+    [0,4,8], #diagonal win
+    [6,4,2], #diagonal win
+    [0,3,6], #row 1 win
+    [1,4,7], #row 2 win
+    [2,5,8], #row 3 win
+    ]
+
+    def initialize(player_1= Players::Human.new, player_2= Players::Human.new("O"), board = Board.new)
+      @board = board
+      @player_1 = player_1
+      @player_2 = player_2
+    end
+
+    def current_player
+      if @board.turn_count.even?
+           player_1
+        else
+           player_2
+      end
+    end
+
+    def over?
+      if @board.full? || won? || draw?
```

Comment:
this should work fine 
```ruby 
def over?
    won? || draw?
end
```

because our draw method is already checking for board to be full